### DISCUSSION TITLE
Is there a way to create a tailwind class for text-align: inherit?

### DISCUSSION THREAD

**Discussion by lifeiscontent** [tailwindlabs/tailwindcss#9909] [Help]
I just came across this issue, because tailwind sets button text to align: center; by default in the reset, so I wanted to set the text alignment to be inherited from a parent (because it changes conditionally) but the issue with the current design of CSS Selectors in tailwind is `tw-text-inherit` refers to `color: inherit;` (it's times like these I wish tailwind classes were more verbose) is there a way to deal with this?

**Comment by wongjn** [comment]
If we look at the core plugins implementation, we can see that there is indeed no `text-align: inherit` utility:
https://github.com/tailwindlabs/tailwindcss/blob/b5f5adf3f5f212ad687118cb854cef8863ed3fae/src/corePlugins.js#L1781-L1790

For your use case, you could look at using an [arbitrary property class](https://tailwindcss.com/docs/adding-custom-styles#arbitrary-properties), `[text-align:inherit]`. Alternatively, you could write a [Tailwind plugin that adds a utility](https://tailwindcss.com/docs/plugins#static-utilities) with this declaration, or [add a class to your CSS](https://tailwindcss.com/docs/adding-custom-styles#adding-custom-utilities) with this declaration.
